### PR TITLE
Fix No notification permission screen sharing on CV

### PR DESCRIPTION
**Pull request by DavDo**
**Jira issue:**
https://glia.atlassian.net/browse/MOB-2929

**What was solved?**

**Release notes:**

 - [ ] Feature
 - [x] Ignore
 - [ ] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

**Additional info:**

 - [x] Tests updated? Added? Unit, acceptance, snapshots?
 - [ ] Logging for future troubleshooting of client issues added?

**Screenshots:**

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizerController.kt
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizerController.kt:1]: Well, these are very sensitive changes. I remember we were discussing them some time ago. Are you convinced in them, are you sure it won't break anything?

Comment (DavDo) [reply widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizerController.kt:1]: I'm not sure. But based on my manual testing results everything works well.

Comment (andrews-moc) [reply widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizerController.kt:1]: Okay. If [these scenarios](https://glia.atlassian.net/wiki/spaces/ENG/pages/3486842894/Dialogs) work fine, then it should be okay.

**Review by andrews-moc — APPROVED** [review]

**Review by Pelkar — APPROVED** [review]
Changes look good, but we need to confirm that no previously working behaviour isn't lost

File: widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizerController.kt
Comment (DavDo) [reply widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizerController.kt:1]: checking...

Comment (DavDo) [reply widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizerController.kt:1]: Passed all the cases above.

Comment (andrews-moc) [reply widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizerController.kt:1]: Okay, great 👍

**Pull request merged**